tfx/benchmarks/beam_pipeline_benchmark_base.py

```diff
@@ -0,0 +1,184 @@
+# Copyright 2019 Google LLC. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import csv
+import os
+os.environ['TF_CPP_MIN_LOG_LEVEL'] = '3'
+import subprocess
+import time
+import yaml
+from absl import flags
+
+from datasets.chicago_taxi import dataset
+from tfma_benchmark_chicago_taxi import TFMABenchmarkChicagoTaxi
+from tfma_v2_benchmark_chicago_taxi import TFMAV2BenchmarkChicagoTaxi
+from tft_benchmark_chicago_taxi import TFTBenchmarkChicagoTaxi
+import benchmark_base
```

Comment:
Use fully qualified imports instead rather than relative imports ([example](https://github.com/tensorflow/tfx/blob/de0e6c7316039ee5f3b2ab3e7814da2da223b8bd/tfx/benchmarks/tfma_benchmark_base.py#L29))

Reply:
Done